test/ng/compileSpec.js

```diff
@@ -7430,3 +7431,19 @@ describe('$compile', function() {
       }));
   });
 });
+
+describe('$compile', function() {
```

Comment:
This is because it is defining the directive in the provide step - it needs to catch the error when the directive is defined.  It cannot lie in the existing top level describe because in the beforeEach, the module is defined then.  It needs to be defined in the spec itself - is it possible to just define a new module and run the spec, so it happens regardless of the beforeEach?